src/domain/engine/Rule/rule.ts

```diff
@@ -0,0 +1,21 @@
+export class Rule {
+  private readonly _agesNumber: number = 3;
+  private readonly _availableWonders: number = 12;
```

Comment:
We need to be clear with the total available wonders and the available wonders by player.
Please rename by **_totalAvailableWonders** and adapt the getter.